commands/annotation.go

```diff
@@ -0,0 +1,69 @@
+// pmm-admin
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"strings"
+
+	"github.com/percona/pmm/api/managementpb/json/client"
+	"github.com/percona/pmm/api/managementpb/json/client/annotation"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+// annotationResult is a result of annotation command.
+type annotationResult struct {
+	Message string `json:"message"`
```

Comment:
It is a rudiment of message from the server. We should do the same as in other commands.

Reply:
(that is, use templates)

Reply:
This implements the Result interface.
```
pmm-admin annotation 'Text' --json
{"message":"Annotation added."}
```
What is your suggestion to use as the key instead of `message`?

Reply:
Nothing. `--json` flag should return data, and we don't have data yet, so it will return `{}`. Success is indicated by exit code 0.

Reply:
Changed.